api-tests/src/test/java/org/openmrs/module/reporting/definition/service/DefinitionServiceTest.java

```diff
@@ -36,12 +36,12 @@ public class DefinitionServiceTest extends BaseModuleContextSensitiveTest {
 	 */
 	@Test
 	public void getDefinitionByUuid_shouldDeserializeCohortIndicatorAndDimensionDataSetDefinition() throws Exception {
-		executeDataSet("org/openmrs/module/reporting/include/DefinitionServiceTest.xml");
-		
-		CohortIndicatorAndDimensionDataSetDefinition persistedDefinition = (CohortIndicatorAndDimensionDataSetDefinition) dataSetDefinitionService
-		        .getDefinitionByUuid("bb1dc014-82a0-4847-8bcd-f74f91282e8d");
-		assertThat(persistedDefinition, notNullValue());
-		assertThat(persistedDefinition.getName(), is("Patients in 2006 by indicators"));
-		assertThat(persistedDefinition.getSpecifications(), not(empty()));
+//		executeDataSet("org/openmrs/module/reporting/include/DefinitionServiceTest.xml");

```

Comment:
What is this?

Reply:
This is a deprecated class. So I am deleting this.